src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -18,8 +21,17 @@
      */
     public DeleteCommand parse(String args) throws ParseException {
         try {
-            Index index = ParserUtil.parseIndex(args);
-            return new DeleteCommand(index);
+            String[] arguments = args.split("\\s+");
+            List<Index> indexes = new ArrayList<>();
+
+            for (String s : arguments) {
+                if (s.length() > 0) {
+                    Index index = ParserUtil.parseIndex(s);
+                    indexes.add(index);
+                }
+            }
+
+            return new DeleteCommand(indexes);
```

Comment:
Would it be better to use argmultimap here rather than implement a custom parser?

Reply:
Have moved this to ParserUtil as discussed